utilities/cleanSearch.js

```diff
@@ -0,0 +1,45 @@
+const checkForAllSpaces = (trend) => {
```

Comment:
for functions that return `true` or `false` like this I personally write the names like `isAllSpaces` because the answer is `true` or `false` or `hasNumber` instead of `checkForNumber` no need to change, just thought I'd share if you find it useful